### ISSUE TITLE
Add generic getClaim() method in ClaimAccessor

### ISSUE THREAD

**Issue by jgrandja** [spring-projects/spring-security#6947]
We should add a convenience (default) method in `ClaimAccessor`, as follows:

```java
default <T> T getClaim(String claim) {
    return (T) this.getClaims().get(claim);
}
```

Related to [comment](https://github.com/spring-projects/spring-security/pull/6799#discussion_r286698589)

**Comment by ThomasVitale** [comment]
Hi @jgrandja, if needed I would be available to work on this issue

**Comment by jgrandja** [comment]
@ThomasVitale That would be great! It should be pretty straight forward - similar to what we already have in `OAuth2AuthorizationRequest.getAttribute()` (in master). Let me know if you have any questions. Thanks.

**Comment by ThomasVitale** [comment]
Hi @jgrandja, I created a PR for the issue, what do you think?

**Comment by codemasterover9000** [comment]
Hi, we just upgraded to Spring Boot from 2.1.1 to 2.2.0. Now we have a failing test for our custom jwtAuthenticationConverter. It seems the implementation of method Jwt.getClaimAsStringList() has changed.

We had a case test where the authorization server sends a space separated list of authorities. In that case getClaimAsStringList() returned null and we did the parsing ourselves. Now it returns a list with the string value of the claim as its only entry. 

This makes it harder to distinguish between an actual list and a space separated string value. Was this intentional?
